design-toolkit/components/src/components/kanban/index.tsx

```diff
@@ -0,0 +1,417 @@
+    >
+      {children}
+    </ol>
+  );
+};
+Col.displayName = 'Kanban.Column';
+
+const ColHeader = ({ children, className, ...rest }: KanbanComponentProps) => {
+  return (
+    <div className={colHeader({ className })} {...rest}>
+      {children}
+    </div>
+  );
+};
+ColHeader.displayName = 'Kanban.Column.Header';
+
+const ColDragHandle = () => (
+  <Icon size='small' aria-label='Drag column'>
+    <DragVert />
+  </Icon>
+);
+ColDragHandle.displayName = 'Kanban.Column.Header.DragHandle';
+
+const ColHeaderTitle = ({
+  children,
+  className,
+  ...rest
+}: KanbanComponentProps) => {
+  return (
+    <span className={colHeaderTitle({ className })} {...rest}>
+      {children}
+    </span>
+  );
+};
+ColHeaderTitle.displayName = 'Kanban.Column.Header.Title';
+
+const ColHeaderActions = ({
+  className,
+  cardCount,
+  children,
+  ...rest
+}: KanbanMenuProps) => {
+  return (
+    <div className={colHeaderActions({ className })} {...rest}>
+      <div className='w-[24px] text-center text-sm'>{cardCount ?? 0}</div>
+      {children}
+    </div>
+  );
+};
+ColHeaderActions.displayName = 'Kanban.Column.Header.Actions';
+
+const ColContent = ({
+  children,
+  className,
+  column,
+  ...rest
+}: KanbanColContentProps) => {
+  const cardIds = column?.cards?.map((card) => card.id) || [];
+
+  return (
+    <SortableContext items={cardIds} strategy={verticalListSortingStrategy}>
+      <div className={colContent({ className })} {...rest}>
+        {children}
+      </div>
+    </SortableContext>
+  );
+};
+ColContent.displayName = 'Kanban.Column.Content';
+
+const ColActions = ({
+  className,
+  onAddCard,
+  ...rest
+}: KanbanColContentActionProps) => {
+  return (
+    <div className={colHeader({ className })} {...rest}>
+      <Button variant='flat' onPress={onAddCard} {...rest}>
+        <Icon>
+          <Add />
+        </Icon>
+        Add Card
+      </Button>
+    </div>
+  );
+};
+ColActions.displayName = 'Kanban.Column.Content.Actions';
+
+// Kanban Card
+function KanbanCard({
+  className,
+  children,
+  card,
+  isActive,
+  ...rest
+}: KanbanCardProps) {
+  const { isDragging, closestEdge, ref, style, attributes, listeners } =
+    useCardInteractions(card);
+  const { activeId } = useDragContext();
+
+  // Check if we should show the drop placeholder
+  const showPlaceholder = activeId && activeId !== card.id && closestEdge;
+
+  return (
+    <li className={cardContainerOuter()} ref={ref} style={style}>
+      {showPlaceholder && closestEdge === 'top' && (
+        <Divider className={divider()} />
+      )}
+
+      <div
+        className={CardInnerStyles({
+          className,
+          isActive,
+          dragging: isDragging,
+        })}
+        data-dragging={isDragging}
+        {...attributes}
+        {...listeners}
+        {...rest}
+      >
+        {children}
+      </div>
+
+      {showPlaceholder && closestEdge === 'bottom' && (
+        <Divider className='mt-s h-xxs bg-accent-primary-bold' />
```

Comment:
I think this should also be (like line 328):

```suggestion
        <Divider className={divider()} />
```